packages/account/src/Components/demo-message/__test__/demo-message.spec.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import DemoMessage from '../demo-message';
+
+// eslint-disable-next-line react/display-name
```

Comment:
This will be fixed [here](https://github.com/binary-com/deriv-app/pull/6306/files#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5R64-R67), No need to ignore it.

Reply:
but I can’t commit with existing warning. husky is blocking it

Reply:
maybe we will be able to delete all such lines in all packages after merging your PR?

Reply:
Alright, let's go with your idea